src/GlobalErrorHandler.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+import { SafeLink, Text, colors } from '@aragon/ui'
+
+class GlobalErrorHandler extends React.Component {
+  state = { error: null }
+  componentDidCatch(error, info) {
+    this.setState({ error })
+  }
+  render() {
+    const { error } = this.state
+    if (!error) {
+      return this.props.children
+    }
+
+    console.log({ err: this.state.error })
+    return (
+      <Main>
+        <In>
+          <Error>
+            <h1>
+              <Text size="xxlarge">An unexpected error has occured</Text>
+            </h1>
+            <Message>{error.message}</Message>
+            <p>
+              Try to <a href="/">reload the application</a>, or{' '}
```

Comment:
Does this send me to the root url (e.g. `app.aragonpm.com`)? If so, it seems unintuitive if I was on a particular DAO / app.